presto-main/src/main/java/com/facebook/presto/execution/QueryStateMachine.java

```diff
@@ -529,6 +528,21 @@ public void setOutput(Optional<Output> output)
         this.output.set(output);
     }
 
+    private void attachConnectorCommitHandleToOutput(ConnectorCommitHandle commitHandle)
```

Comment:
nit: The name isn't quite accurate since this isn't attaching the commit handle, it's attaching the serialized commit output.  Maybe change it to something like `addSerializedCommitOutputToOutput`